lib/site.js

```diff
@@ -1,17 +1,18 @@
 /**
  * Module dependencies.
  */
-import Post from './site.post';
 import Category from './site.category';
-import Tag from './site.tag';
-import Media from './site.media';
 import Comment from './site.comment';
-import SiteWordAds from './site.wordads';
 import Follow from './site.follow';
-import SitePlugin from './site.plugin';
+import Media from './site.media';
```

Comment:
Yes, but maybe in another PRs? I just wanted sort the current ones by name. I can create a specific PR for that.